test/solc/CommandLineParser.cpp

```diff
@@ -250,7 +250,7 @@ BOOST_AUTO_TEST_CASE(no_import_callback)
 		{"solc", "--strict-assembly", "--no-import-callback", "input.yul"},
 		{"solc", "--import-ast", "--no-import-callback", "ast.json"},
 		{"solc", "--link", "--no-import-callback", "input.bin"},
-		{"solc", "--yul", "--no-import-callback", "input.yul"},
+		{"solc", "--no-import-callback", "input.yul"},
```

Comment:
I think you can remove this line, since it will fail without `--strict-assembly` option and there is already an entry testing it.